model/src/main/proto/profile.proto

```diff
@@ -8,7 +8,7 @@ option java_multiple_files = true;
 // Top level proto used to store last used id and all profiles.
 message ProfileDatabase {
   // Represents the last used unique ID for adding a profile.
```

Comment:
Update documentation to match new field name.

Reply:
Updated.